README.md

```diff
@@ -33,16 +33,18 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your Team Lead
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
-
-2. Name two big differences between ```display: block;``` and ```display: inline;```.
-
+   semantic html is the use of tags that is descriptive of the content
```

Comment:
Very true, it is also very important for SEO and accessibility ->https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML